srcpkgs/zenity/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'zenity'
 pkgname=zenity
-version=3.44.3
+version=4.0.1
 revision=1
 build_style=meson
-configure_args="$(vopt_bool webkit webkitgtk)"
-hostmakedepends="gettext itstool perl pkg-config gtk-update-icon-cache"
-makedepends="gtk+3-devel libglib-devel libnotify-devel $(vopt_if webkit libwebkit2gtk41-devel)"
+configure_args="-Dwebkitgtk=$(vopt_if webkit true false)"
+hostmakedepends="gettext glib-devel gtk-update-icon-cache help2man itstool
+ pkg-config"
+makedepends="$(vopt_if webkit libwebkitgtk60-devel) libadwaita-devel"
 short_desc="Display GNOME dialogs from the command line"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="LGPL-2.0-or-later"
 homepage="https://help.gnome.org/users/zenity/"
-changelog="https://gitlab.gnome.org/GNOME/zenity/-/raw/zenity-3-44/NEWS"
+changelog="https://download.gnome.org/sources/zenity/4.0/zenity-4.0.1.news"
```

Comment:
This should point to `https://gitlab.gnome.org/GNOME/zenity/-/raw/master/NEWS`

Reply:
fixed